Sources/BraveWallet/Crypto/Stores/SendTokenStore.swift

```diff
@@ -165,8 +168,14 @@ public class SendTokenStore: ObservableObject {
         network: network,
         decimalFormatStyle: .decimals(precision: Int(selectedSendToken.decimals))
       )
+  
+      let metadataId = selectedSendToken.id + selectedSendToken.chainId
```

Comment:
`selectedSendToken.id` already includes `chainId`.

Reply:
fixed in [00c5f80](https://github.com/brave/brave-ios/pull/6534/commits/00c5f80308e952e4dc6914ae61d3bbcddda7d726)